src/Card.tsx

```diff
@@ -52,6 +52,30 @@ export function Card<T extends ElementType = "div">({
     ? { target: "_blank", rel: "noreferrer" }
     : {};
 
+  let testIconisImage: boolean = false;
+
+  if (typeof icon === "string") {
+    testIconisImage = icon.startsWith("http://") || icon.startsWith("https://");
+  }
+
+  const renderIcon: JSX.Element = (
+    <>
+      {icon ? (
+        testIconisImage ? (
+          <img
+            src={icon as string}
+            alt={title}
+            className="h-6 w-6 rounded-full object-cover object-center"
```

Comment:
The `rounded-full` looks good but lets remove it for now. We need to let people display square images if they want to.